cms/djangoapps/contentstore/rest_api/v0/views/tabs.py

```diff
@@ -198,24 +198,12 @@ def post(self, request: Request, course_id: str) -> Response:
         Move course tabs:
 
             POST /api/contentstore/v0/tabs/{course_id}/reorder [
-                {
-                    "tab_id": "info"
-                },
-                {
-                    "tab_id": "courseware"
-                },
```

Comment:
@awaisdar001 
1. Yes. The frontend will need to specify the new position of all tabs. 
2. This was a new endpoint we had added recently. So it isn't in use in any MFE yet. It's also why I went for a `v0` in the API since I expected bigger changes to this API in the short term.

Reply:
Thanks @xitij2000